cfg/train_fc.yaml

```diff
@@ -0,0 +1,139 @@
+# general optimization params
```

Comment:
Can you change the name of this file to train_fc_pj.yaml? I want it to be consistent if possible.